website/docs/zkvm/developer-guide/optimization.md

```diff
@@ -0,0 +1,436 @@
+# Guest Optimization Guide
+
+RISC Zero’s zkVM is designed and built to act like a physical CPU.
+We did this so you could use general purpose languages (e.g. Rust) and general purpose tools (e.g. Cargo, LLVM). **Generally, you can apply general purpose optimization techniques too!**
+
+In this guide, we’ll cover optimization of [zkVM guest programs].
+We’ll discuss when you can apply common techniques from other contexts, and when you need a new approach.
+
+[zkVM guest programs]: ./guest-code-101
+
+## Background
+
+### What is the zkVM, really?
+
+**The zkVM is essentially a CPU.**
+
+In particular, it is an implementation of the [RISC-V architecture] (specifically riscv32im), similar to how your laptop’s CPU may be an implementation of an [x86] or [ARM] architecture.
+This biggest difference is that the zkVM is implemented with [arithmetic circuits], in software, instead of circuitry made from silicon and copper.
+
+[RISC-V architecture]: /reference-docs/about-risc-v
+[x86]: https://en.wikipedia.org/wiki/X86
+[ARM]: https://en.wikipedia.org/wiki/ARM_architecture_family
+[arithmetic circuits]: https://dev.risczero.com/proof-system/stark-by-hand
+
+### What is a “cycle”?
+
+Both in the zkVM and on physical CPUs, the cost of an operation is measured in “clock cycles”.
+
+Intuitively, a "clock cycle" is the smallest unit of time in a CPU's operation, representing one tick of the CPU's internal clock and the time it takes to execute a basic CPU operation (e.g. adding two integers).
+We’ll refer to this as a “cycle”. [^1]
+
+**Proving times for the zkVM are directly related to the number of cycles in an execution.**
```

Comment:
Seems like a nice spot to include a link to our datasheet for more information

Reply:
My current understanding is we do not have a version of that sheet that we want to share widely right now. If we do, I definitely want to link to it.